pinot-tools/pom.xml

```diff
@@ -165,6 +163,7 @@
                 <extraArguments>
                   <extraArgument>-XX:MaxDirectMemorySize=30g</extraArgument>
                   <extraArgument>-Dlog4j2.configurationFile=conf/pinot-tools-log4j2.xml</extraArgument>
+                  <extraArgument>-Dplugins.dir=plugins</extraArgument>
```

Comment:
can we make it absolute? I think there is a variable set in the script to get the basedir